pom.xml

```diff
@@ -134,6 +134,7 @@
         <spotify.version>8.15.1</spotify.version>
 
         <!-- Plugins versions -->
+        <buildnumber-maven-plugin.version>3.0.0</buildnumber-maven-plugin.version>
```

Comment:
Optional, just OCD, if you want to move this down to 157 so they are grouped for similar name and sorted